ush/job_preamble.sh

```diff
@@ -161,19 +161,11 @@ fi
 #-----------------------------------------------------------------------
 #
 function job_postamble() {
-
     # Remove temp directory
-    if [ "${RUN_ENVIR}" = "nco" ] && [ "${KEEPDATA}" = "FALSE" ]; then
+    if [ "${RUN_ENVIR}" = "nco" ] && [ "${KEEPDATA}" = "NO" ]; then
+        echo "DATA clean up for directory: ${DATA}"
```

Comment:
@lgannoaa, this change makes KEEPDATA:false not work correctly.

Reply:
The job_postamble function solution can result in NCO unable to recover failed cycle during some production emergency condition. 
It is replaced by dedicated data_cleanup job designed. Please read the justification above.